ELCodableTests/ELCodableTests.swift

```diff
@@ -138,7 +138,7 @@ class ELCodableTests: XCTestCase {
         let model = try? TestModel.decode(json)
         print(model as Any)
         
-        let output = try? model?.encode()
+        let output = ((try? model?.encode()) as JSON??)
```

Comment:
Can we use Swift 5's flattened optional here ?
https://github.com/apple/swift-evolution/blob/master/proposals/0230-flatten-optional-try.md

Reply:
Maybe? I was trying to keep changes minimal in this lib since it's deprecated but I can take a look.

Reply:
Oh right. I think it wouldn't make sense for deprecated library.